package-lock.json

```diff
@@ -0,0 +1,1263 @@
+      "funding": [
+        {
+          "type": "github",
+          "url": "https://github.com/sponsors/csstools"
+        },
+        {
+          "type": "opencollective",
+          "url": "https://opencollective.com/csstools"
+        }
+      ],
+      "engines": {
+        "node": ">=18"
+      },
+      "peerDependencies": {
+        "postcss-selector-parser": "^7.0.0"
+      }
+    },
+    "node_modules/@csstools/selector-specificity": {
+      "version": "5.0.0",
+      "resolved": "https://registry.yarnpkg.com/@csstools/selector-specificity/-/selector-specificity-5.0.0.tgz",
+      "integrity": "sha1-A3gXtXQmITTKvWj8TsGkVPFoQHs= sha512-PCqQV3c4CoVm3kdPhyeZ07VmBRdH2EpMFA/pd9OASpOEC3aXNGoqPDAZ80D0cLpMBxnmk0+yNhGsEx31hq7Gtw==",
+      "funding": [
+        {
+          "type": "github",
+          "url": "https://github.com/sponsors/csstools"
+        },
+        {
+          "type": "opencollective",
+          "url": "https://opencollective.com/csstools"
+        }
+      ],
+      "engines": {
+        "node": ">=18"
+      },
+      "peerDependencies": {
+        "postcss-selector-parser": "^7.0.0"
+      }
+    },
+    "node_modules/@fortawesome/fontawesome-free": {
+      "version": "7.0.1",
+      "resolved": "https://registry.yarnpkg.com/@fortawesome/fontawesome-free/-/fontawesome-free-7.0.1.tgz",
+      "integrity": "sha1-wax/B7ot9H0d57cjb60lxObKUHY= sha512-RLmb9U6H2rJDnGxEqXxzy7ANPrQz7WK2/eTjdZqyU9uRU5W+FkAec9uU5gTYzFBH7aoXIw2WTJSCJR4KPlReQw==",
+      "engines": {
+        "node": ">=6"
+      }
+    },
+    "node_modules/@hotwired/turbo": {
+      "version": "8.0.13",
+      "resolved": "https://registry.yarnpkg.com/@hotwired/turbo/-/turbo-8.0.13.tgz",
+      "integrity": "sha1-qzX9qdNYQyyKhyqDOESzjMy4wls= sha512-M7qXUqcGab6G5PKOiwhgbByTtrPgKPFCTMNQ52QhzUEXEqmp0/ApEguUesh/FPiUjrmFec+3lq98KsWnYY2C7g==",
+      "engines": {
+        "node": ">= 14"
+      }
+    },
+    "node_modules/@hotwired/turbo-rails": {
+      "version": "8.0.16",
+      "resolved": "https://registry.yarnpkg.com/@hotwired/turbo-rails/-/turbo-rails-8.0.16.tgz",
+      "integrity": "sha1-CW877Iw8BNSqxN0PdTBcZ7PbNQE= sha512-Yxiy2x+N3eOIEDokvLzSrd08aI5RDKnFYDQFl2J/LuMEWTtPdY7oNP0F/gv/sSe5AV23Lwz4FitG/uNFXNM5tA==",
+      "dependencies": {
+        "@hotwired/turbo": "^8.0.13",
+        "@rails/actioncable": ">=7.0"
+      }
+    },
+    "node_modules/@lydell/node-pty": {
+      "version": "1.0.2",
+      "resolved": "https://registry.yarnpkg.com/@lydell/node-pty/-/node-pty-1.0.2.tgz",
+      "integrity": "sha1-f4IN2RfCfvJ8uEbLZoTpBbOR3is= sha512-5UhusYFuH8GDjmF0+WKAHYv6HrB+AhuGBIcSFHOVRLC4aCf1JMTbrnOtZvXKPQV4MeBupZ9EzN6IZbjvjfI1Ew==",
+      "dev": true,
+      "optionalDependencies": {
+        "@lydell/node-pty-darwin-arm64": "1.0.2",
+        "@lydell/node-pty-darwin-x64": "1.0.2",
+        "@lydell/node-pty-linux-x64": "1.0.2",
```

Comment:
The lock file doesn't contain these optional dependencies which caused me, at least, to to be ably to run `run-pty`:

```
/home/matt/projects/st-pax/node_modules/@lydell/node-pty/requireBinary.js:12
            throw new Error(optionalDependencies === undefined
            ^

Error: The @lydell/node-pty package supports your platform (linux-x64), but it could not find the binary package for it: @lydell/node-pty-linux-x64/pty.node

This can happen if you use the "--omit=optional" (or "--no-optional") npm flag.
The "optionalDependencies" package.json feature is used to install the correct
binary executable for your current platform. Remove that flag to use @lydell/node-pty.
```